presto-docs/src/main/sphinx/admin/verifier.rst

```diff
@@ -286,6 +288,9 @@ Name                                        Description
 ``running-mode``                            Set to ``query-bank`` to make the Verifier run in ``query-bank`` mode. Supports
                                             ``query-bank`` and ``control-test``. Defaults to ``control-test``.
 ``save-snapshot``                           Set to ``true`` to save checksums to ``mysql`` database.
+``nondeterministic-function-substitutes``   Specification of substituting nondeterministic functions with deterministic functions,
```

Comment:
Let's also clarify that substitution function must have compatible signature, e.g. accept a subset of inputs and produce same return type.